dashboard/src/features/accessControl/constants.js

```diff
@@ -20,3 +20,15 @@ export const policyOptions = [
     hint: 'Access to monitoring-specific endpoints'
   },
 ]
+
+export const subjectFieldOptions = [
```

Comment:
Okay, real talk. When I put it as `CN`, the SDK converted the key to `_c_n`, which is no good. Should our snake-caser ignore strings that are all uppercase?